tests/foreman/ui/test_environment.py

```diff
@@ -34,17 +36,13 @@ def test_create_env_positive_1(self, name):
 
         """
         with Session(self.browser) as session:
-            make_env(session, name=name)
-            search = self.environment.search(name)
-            self.assertIsNotNone(search)
-
-    @data(
-        gen_string('alpha', 255),
-        gen_string('numeric', 255),
-        gen_string('alphanumeric', 255)
-    )
```

Comment:
There is no need to test 255 characters for alpha/numeric/alphanumeric. I just used alphanumeric now in test_create_env_positive_2